src/Plugin/resource/DataProvider/DataProviderEntity.php

```diff
@@ -1146,6 +1147,10 @@ protected function getReferencedId($value, ResourceFieldInterface $resource_fiel
       $target_entity_type = $field_info['settings']['target_type'];
       $bundles = empty($field_info['settings']['handler_settings']['target_bundles']) ? array() : $field_info['settings']['handler_settings']['target_bundles'];
     }
+    elseif (!empty($field_info['type']) && $field_info['type'] == 'commerce_product_reference') {
```

Comment:
Is this duplicated code? If so, we should have an `||` condition in the `if` clause instead of duplicating.

Reply:
You're right!! 👍